src/js/components/DataTableColumns/__tests__/DataTableColumns-test.tsx

```diff
@@ -38,6 +39,34 @@ describe('DataTableColumns', () => {
 
     expect(getByRole('button', { name: 'Open column selector' })).toBeTruthy();
     expect(container.firstChild).toMatchSnapshot();
+
+    fireEvent.click(
+      screen.getByRole('button', { name: 'Open column selector' }),
+    );
+
+    window.scrollTo = jest.fn();
+
+    // advance timers so drop can open
+    act(() => jest.advanceTimersByTime(200));
+    // Take a snapshot of the tab panel
+    const firstTabPanel = screen.getByRole('tabpanel', {
+      name: 'Select columns Tab Contents',
+    });
+    expect(firstTabPanel).toMatchSnapshot();
```

Comment:
Do we have to take a screenshot of the panel directly or can we simply to `expect(asFragment()).toMatchSnapshot();`?

Reply:
if we just have 
`expect(asFragment()).toMatchSnapshot();`

This will just take a snapshot of the following so it wont take the snap shot of when what is actually inside the Select columns Tab content just the icon that opens the DataTableColumns
```
 <Data id="test-data" data={data}>
          <DataFilters>
            <DataTableColumns drop options={['name', 'size']} />
          </DataFilters>
          <DataTable
            columns={[
              { property: 'name', header: 'Name' },
              { property: 'size', header: 'Size' },
            ]}
          />
        </Data>

```